apps/spruce/src/pages/adminSettings/tabs/AnnouncementsTab/AnnouncementTab.tsx

```diff
@@ -1,16 +1,30 @@
 import styled from "@emotion/styled";
-import { H2, H3 } from "@leafygreen-ui/typography";
+import { H2 } from "@leafygreen-ui/typography";
 import { size } from "@evg-ui/lib/constants/tokens";
+import { AdminSettingsTabRoutes } from "constants/routes";
+import { BaseTab } from "../BaseTab";
+import { getFormSchema } from "./formSchema";
+import { TabProps } from "./types";
 
-export const AnnouncementTab = () => (
-  <div>
-    <TitleContainer>
-      <H2>Announcements</H2>
-    </TitleContainer>
-    <H3>Announcements</H3>
-    <p>Announcements settings for the application.</p>
-  </div>
-);
+const tab = AdminSettingsTabRoutes.Announcements;
```

Comment:
idk why we do this pattern in project settings, can you just pass this directly into `BaseTab` without declaring a variable

Reply:
yies it works, thank you